packages/form-js-editor/src/features/properties-panel/entries/LabelEntry.js

```diff
@@ -51,7 +51,12 @@ export default function LabelEntry(props) {
       editField,
       field,
       isEdited: isFeelEntryEdited,
-      isDefaultVisible: (field) => INPUTS.includes(field.type) || field.type === 'button' || field.type === 'group'
+      isDefaultVisible: (field) => (
+        INPUTS.includes(field.type) ||
```

Comment:
Might make sense to have a parameter for "labeled" at this point, but I don't know. Just a thought for the future, since this conditional is starting to get more and more arbitrary.